internal/console/kubevirt_target.go

```
@@ -0,0 +1,157 @@
+	transportConfig, err := config.TransportConfig()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get transport config: %w", err)
+	}
+
+	capture := &headerCaptureRoundTripper{}
+	rt, err := transport.HTTPWrappersForConfig(transportConfig, capture)
+	if err != nil {
+		return nil, fmt.Errorf("failed to build auth wrappers: %w", err)
+	}
+
+	req, err := http.NewRequest(http.MethodGet, "https://placeholder", nil)
```

golangci-lint (2.12.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)